BackofficeBundle/Resources/public/less/smartadminPatches/widget.less

```diff
@@ -1,10 +1,10 @@
 .js-widget-title {
     margin-left: 10px;
     display: inline-block;
-    max-width: 90%;
```

Comment:
why remove this line ? this line is used when a the navigator don't support calc property. Like IE9 where calc property isn't totally supported